src/pages/studies/StudyTableContainer.tsx

```diff
@@ -23,11 +23,20 @@ const mapState = (state: RootState): StudyTableContainerState => ({
 const connector = connect(mapState, mapDispatch);
 
 type PropsFromRedux = ConnectedProps<typeof connector>;
+export type PaginationType = {
+  pagination: {
+    current: number;
+    pageSize: number;
+    total: number;
+    onChange: (page: number) => void;
+  };
+};
 
-type Props = StudiesResults & PropsFromRedux;
+type Props = StudiesResults & PropsFromRedux & PaginationType;
 
 const StudyTable: FC<Props> = (props) => {
-  if (props.loading) {
+  const { loading, pagination } = props;
+  if (loading) {
     return null;
```

Comment:
don't we want to display a <Spin/> or else?

Reply:
Unrelated to the change.
But now one state when empty and it keeps the previous data while loading as spected.